docs/rules/jsx-uses-react.md

```diff
@@ -1,6 +1,6 @@
 # Disallow React to be incorrectly marked as unused (`react/jsx-uses-react`)
 
-💼 This rule is enabled in the `recommended` [config](https://github.com/jsx-eslint/eslint-plugin-react/#shareable-configs). This rule is _disabled_ in the `jsx-runtime` [config](https://github.com/jsx-eslint/eslint-plugin-react/#shareable-configs).
+💼🚫 This rule is enabled in the ☑️ `recommended` [config](https://github.com/jsx-eslint/eslint-plugin-react/#shareable-configs). This rule is _disabled_ in the `jsx-runtime` [config](https://github.com/jsx-eslint/eslint-plugin-react/#shareable-configs).
```

Comment:
is it weird the "disabled" emoji appears on a line where it talks about what's enabled? i might expect this:
```suggestion
💼 This rule is enabled in the ☑️ `recommended` [config](https://github.com/jsx-eslint/eslint-plugin-react/#shareable-configs).
🚫 This rule is _disabled_ in the `jsx-runtime` [config](https://github.com/jsx-eslint/eslint-plugin-react/#shareable-configs).
```
instead?

Reply:
That's an option, although easier would be to just move the emoji inside the notice so it's next to the sentence it refers to:

💼 This rule is enabled in the ☑️ `recommended` [config](https://github.com/jsx-eslint/eslint-plugin-react/#shareable-configs). 🚫 This rule is _disabled_ in the `jsx-runtime` [config](https://github.com/jsx-eslint/eslint-plugin-react/#shareable-configs).

Either your suggestion or mine would both work?

Reply:
I filed https://github.com/bmish/eslint-doc-generator/issues/310 to track this. I'll address it but I don't think we need to block this PR on it.

Reply:
What I like about separate lines is that all the emojis align left.

Agreed on not blocking this PR.

Reply:
Will discuss in that issue.